lib/generators/shopify_app/shop_model/shop_model_generator.rb

```diff
@@ -16,6 +16,26 @@ def create_shop_migration
         migration_template('db/migrate/create_shops.erb', 'db/migrate/create_shops.rb')
       end
 
+      def create_shop_with_access_scopes_migration
+        scopes_column_prompt = <<~PROMPT
+          It is highly recommended that apps record the access scopes granted by \
+          merchants during app installation. See app/models/shop.rb to modify how \
+          access scopes are stored and retrieved.
+
+          [WARNING] Your app will fail the ScopesVerification concern if access scopes are not stored.
```

Comment:
Does anything use this by default? If so, we may want to mention that in the message as well, so we don't 'scare' people into accepting it?

Reply:
I can provide a better reason for it.